pkg/cloudprovider/aws/fleet/packing/packing.go

```diff
@@ -38,19 +42,141 @@ type Packings struct {
 	InstanceTypeOptions []string
```

Comment:
nit: I'd just call this `InstanceTypes` or maybe `AcceptableInstanceTypes` - "options" makes it sound like it's some kind of additional options about the instances (like whether or to delete EBS volumes on detach, or something like that).